Sources/Build/BuildPlan.swift

```diff
@@ -1782,7 +1794,7 @@ public class BuildPlan {
 
         pkgConfigCache[target] = ([String](cflagsCache), libsCache)
 
-        return pkgConfigCache[target]!
+        return pkgConfigCache[target]! // forced unwrap safe
```

Comment:
For these I usually tend to do the more verbose thing of creating it to a local variable, then storing it in the dictionary, and then returning the local variable.  It's more verbose but avoids having to explain a force unwrap.  But it's of course a matter of preference.

Reply:
> For these I usually tend to do the more verbose thing of creating it to a local variable, then storing it in the dictionary, and then returning the local variable

not sure I follow your technique, can you give an example?

Reply:
oh I see... before setting it in the "cache" you create a local variable. makes sense

Reply:
Yeah, I just really dislike force unwraps...  so am willing to have more verbiage.  But that's probably just a question of style.

Reply:
```
let foo = bar()
baz[something] = foo
return foo
`

Reply:
add a fixup to adopt this. ptal

Reply:
Looks great.  Thanks!